daemon/userconsent.go

```diff
@@ -0,0 +1,173 @@
+//go:build moose
```

Comment:
I'd avoid adding build flags here and rather move them to the `main` package as this does not actually depend on moose but rather is a new checker that can be used with or without moose.

Reply:
Done